src/main/java/pl/shop/service/impl/ProductCreateServiceImpl.java

```diff
@@ -21,10 +21,11 @@
     private final ProductMapper productMapper;
 
     @Override
-    public ProductDto save(ProductDto productDto) {
+    public ProductDto createProduct(ProductDto productDto) {
         productDto.setCreatedAt(new Date());
 
-        Product savedProduct = productRepository.save(productMapper.productDtoToProduct(productDto));
+        Product receivedMappedProduct = productMapper.productDtoToProduct(productDto);
+        Product savedProduct = productRepository.save(receivedMappedProduct);
```

Comment:
Wydaje mi się to bardziej czytelne teraz, jakieś pomysły ?

Reply:
Mam z tyłu głowy, że zmieniliśmy nazwę, bo ja ją myliłem z metodą z CrudRepository. :> No dla mnie jest lepiej, trudniej mi będzie nie zczaić co się wydarza w kodzie.